.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,37 @@
+name: integration-test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  integration-test:
+    name: Run Integration- Test
+    runs-on: ubuntu-latest
+    environment: Integration Test
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
```

Comment:
Also is kind able to pick up on these creds? I feel we may have to do some extra steps.

Reply:
It worked on the [private branch](https://github.com/astaticvoid/aws-cloud-map-mcs-controller-for-k8s/runs/4162413311?check_suite_focus=true)

Reply:
great!